_config.yml

```diff
@@ -161,6 +161,10 @@ release_archive:
         version_short: "1.13"
         version_long: 1.13.0
         release_date: 2021-04-30
+      -
+        version_short: "1.12"
+        version_long: 1.12.5
+        release_date: 2021-07-02
```

Comment:
Please remember to update the release date and also the file **2021-07-30-release-1.12.5.md** during merging the PR.